about/package-scope.md

```diff
@@ -30,9 +28,229 @@ open a [pre-submission inquiry using a GitHub Issue](https://github.com/pyOpenSc
+pyOpenSci domain scope. Note that your package should have some level of
+demonstrated scientific application. This could be a use case that you can
+link to or a tutorial that demonstrates it's potential application for science.
+
+Below we provide examples of packages from pyOpenSci ecosystem. Because we
+have growing community of packages, in some cases we link to R packages
+within the rOpenSci community that match the category scope for reference.
+
+We will update this page as our review process evolves.
+
+```{note}
+Many of the example packages below perform tasks that might fit in multiple
+categories. There are just there to provide you with a flavor of the types
+of packages that would fall into that category.
+```
+
+### Data retrieval
+Packages for accessing and downloading data from online sources. This category
+includes wrappers for accessing APIs.
+
+Our definition of scientific applications is broad, including data storage
+services, journals, and other remote servers, as many data sources may be of
+interest to scientists. However, retrieval packages should be focused on data
+sources / topics, rather than services.
+
+*For example a general client for Amazon Web Services data storage would not be in-scope.*
+
+* Examples: [OpenOmics](https://github.com/pyOpenSci/software-submission/issues/31)
+
+
+### Data extraction
+
+These packages aid in retrieving data from unstructured sources such as text,
+images, and PDFs. They might also parse scientific data types and outputs from
+scientific equipment.
+
+* Examples: [devicely](https://github.com/pyOpenSci/software-submission/issues/37), [jointly](https://github.com/pyOpenSci/software-submission/issues/45)
+
+### Data processing & munging
+
+Data munging tools are tools that support processing data discussed above. This
+category focuses on tools for handling data in specific  formats that scientists
+may be interested in working with. These data may also be generated from
+scientific workflows or exported from instruments and wearables.
+
+* Examples: [physcraper](https://github.com/pyOpenSci/software-submission/issues/26)
+
+
+### Data deposition
+
+Tools for depositing data into scientific research repositories.
+
+* Examples: [This is an example from rOpenSci - eml](https://github.com/ropensci/software-review/issues/80)
+
+### Data validation and testing:
+
+Tools that enable automated validation and checking of data quality and
+completeness. These tools should be able to support scientific workflows.
+
+* Example: [pandera](https://github.com/pyOpenSci/software-submission/issues/12)
+
+### Scientific software wrappers
+
+Scientific software wrappers refer to packages that provide a Python interface
+for existing scientific packages written in other languages.
+
+These programs must be specific to scientific domains and not general computing
+utilities. Wrappers must provide significant added value to the scientific
```

Comment:
"not general computing utilities" is a little awkward to me. What is a general computing utility? Compression? Database ORM or connector? NumPy/pandas/xarray? I like your example above about general AWS client is out of scope. A general tool like compression, though, can be primarily developed and useful for scientific applications. So, I think this is a fine needle to try to thread.

"not general computing utilities" clause _may_ be okay as is, b/c I don't know how to improve it. Let me illustrate why this is awkward. If a data utility is created by scientists to serve their specific needs, but they decide to do a little more work to make it _generic_, isn't that a good thing to be rewarded?

Heck, maybe "must be specific to scientific domains" is also too strong. Perhaps something like "must be essential to".

(I'm probably over-thinking this).

Reply:
i think if you are "over thinking" (which you are not) / questioning then someone else will too. What if we removed the general computing part and just said something like... 

these programs should have clear application to scientific applications?

I think what we don't want is some python utility tool that is cool but also somethign that a scientist would likely not ever use / find useful. SO maybe language around demonstrated or clear application to open science is better language?

Reply:
I like "these programs should have clear application to scientific applications?" :)

Reply:
awesome! ok i'll fix this. many thanks! 👐

Reply:
ok i have this now - 

_These packages should have a clear scientific application. Wrappers must provide 
significant added value to the scientific ecosystem be it in data handling, or 
improved installation processes for Python users._

I know that ropensci does have language around general data utilities NOT being in scope but here i am having a hard time defining what that means. so perhaps we start with a clear scientific application and as edge cases come up we can re-evaluate and adjust language!!